apps/web/components/Shell.tsx

```diff
@@ -201,218 +198,222 @@ export default function Shell(props: {
+                  )}>
+                  {props.children}
+                </div>
+                {/* show bottom navigation for md and smaller (tablet and phones) */}
+                {status === "authenticated" && (
+                  <nav className="bottom-nav fixed bottom-0 z-30 flex w-full bg-white shadow md:hidden">
+                    {/* note(PeerRich): using flatMap instead of map to remove settings from bottom nav */}
+                    {navigation.flatMap((item, itemIdx) =>
+                      item.href === "/settings/profile" ? (
+                        []
+                      ) : (
+                        <Link key={item.name} href={item.href}>
+                          <a
                             className={classNames(
-                              item.current ? "text-gray-900" : "text-gray-400 group-hover:text-gray-500",
-                              "mx-auto mb-1 block h-5 w-5 flex-shrink-0 text-center"
+                              item.current ? "text-gray-900" : "text-neutral-400 hover:text-gray-700",
+                              itemIdx === 0 ? "rounded-l-lg" : "",
+                              itemIdx === navigation.length - 1 ? "rounded-r-lg" : "",
+                              "group relative min-w-0 flex-1 overflow-hidden bg-white py-2 px-2 text-center text-xs font-medium hover:bg-gray-50 focus:z-10 sm:text-sm"
                             )}
-                            aria-hidden="true"
-                          />
-                          <span className="truncate">{item.name}</span>
-                        </a>
-                      </Link>
-                    )
-                  )}
-                </nav>
-              )}
-              {/* add padding to content for mobile navigation*/}
-              <div className="block pt-12 md:hidden" />
-            </div>
-            <LicenseBanner />
-          </main>
+                            aria-current={item.current ? "page" : undefined}>
+                            <item.icon
+                              className={classNames(
+                                item.current ? "text-gray-900" : "text-gray-400 group-hover:text-gray-500",
+                                "mx-auto mb-1 block h-5 w-5 flex-shrink-0 text-center"
+                              )}
+                              aria-hidden="true"
+                            />
+                            <span className="truncate">{item.name}</span>
+                          </a>
+                        </Link>
+                      )
+                    )}
+                  </nav>
+                )}
+                {/* add padding to content for mobile navigation*/}
+                <div className="block pt-12 md:hidden" />
+              </div>
+              <LicenseBanner />
+            </main>
+          </div>
         </div>
-      </div>
-    </>
-  );
+      </>
+    );
+
+  return null;
```

Comment:
We return null when there's no `session`. This occurs when `session=null` and `loading=false`. The `useEffect` which redirects to `/auth/login` is run after this render.

We can also choose to show loader in this case.